app/services/statement_of_cases/pdf_converter.rb

```diff
@@ -0,0 +1,35 @@
+module StatementOfCases
+  class PdfConverter
+    def self.call(statement_of_case)
+      new(statement_of_case).call
+    end
+
+    attr_reader :statement_of_case
+
+    def initialize(statement_of_case)
+      @statement_of_case = statement_of_case
+    end
+
+    def call
+      statement_of_case.pdf_file.attach(
+        io: File.open(converted_file.path),
+        filename: "#{original_file_base_name}.pdf",
+        content_type: 'application/pdf'
+      )
+    end
+
+    private
+
+    def converted_file
+      ActiveStoragePdfConverter.call(statement_of_case.original_file)
+    end
+
+    def original_file_base_name
+      File.basename(original_file_name, File.extname(original_file_name))
+    end
+
+    def original_file_name
+      @original_file_name ||= statement_of_case.original_file.filename.to_s
```

Comment:
Is 'filename' going to ever be anything other than a string ... and if it's not a string shouldn't that cause an error rather than just returning a string representation of that not-a-string thing.

Reply:
`filename` is actually an instance of `ActiveStorage::Filename` (https://api.rubyonrails.org/classes/ActiveStorage/Filename.html).
But I just realised that it has `.base` and `.extension_with_delimiter` methods so I should be able to use that instead of `File.basename` and `File.extname` on line 28 of this file